src/entities/note.cpp

```diff
@@ -2053,6 +2053,18 @@ void highlightCode(QString &str, const QString &type, int cbCount) {
     }
 }
 
+static int nonOverlapCount(const QString &str, const QChar c = '`') {
```

Comment:
Did you put this method *not* into the header file on purpose?

Reply:
yes, its not being used anywhere, so i wanted to limit this to note.cpp only

Reply:
does this fix the crash on your side?

Reply:
will test  when I have access to my Manjaro box

Reply:
Crash gone on Manjaro, great job!